indexer/indexer_tests/osm_editor_test.hpp

```diff
@@ -0,0 +1,81 @@
+#pragma once
+
+#include "generator/generator_tests_support/test_feature.hpp"
+#include "generator/generator_tests_support/test_mwm_builder.hpp"
+
+#include "platform/local_country_file_utils.hpp"
+
+#include "storage/country_info_getter.hpp"
+
+#include "indexer/mwm_set.hpp"
+
+#include "map/feature_vec_model.hpp"
+
+namespace tests
+{
+class EditorTest
+{
+public:
+  EditorTest();
+  ~EditorTest();
+
+  void GetFeatureTypeInfoTest();
```

Comment:
Do you really need Test suffix for methods in the test class in the test folder? Is it really informative?